assets/custom.js

```diff
@@ -10,8 +10,16 @@ function activate_sidebar() {
 function activate_dropdown() {
```

Comment:
this implementation is so wrong, bro... if you want to declare a function, don't trigger it when window is ready. instead, if you want the code triggered when your window is ready, don't declare it as a function.